src/styles/_catalog.scss

```diff
@@ -1,22 +1,35 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  $card-width: 200px;
+
+  display: grid;
+  grid-template-columns: repeat(4, $card-width);
+  gap: 48px;
+  justify-content: center;
 
   max-width: 944px;
   margin: 0 auto;
```

Comment:
you should have used either `justify-content: center` or `margin: 0 auto`, they cause similar effect